UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/NoticestoMarinersController.cs

```diff
@@ -30,7 +30,7 @@ public async Task<IActionResult> Index()
             {
                 ViewBag.IsDistributor = _userService.IsDistributorUser;
 
-                _logger.LogInformation(EventIds.Start.ToEventId(), "Maritime safety information request to get weekly NM files started for correlationId:{correlationId}", GetCurrentCorrelationId());
+                _logger.LogInformation(EventIds.Start.ToEventId(), "Maritime safety information request to get weekly NM files started for User:{_userService.SignInName} and IsDistributor:{_userService.IsDistributorUser} and correlationId:{correlationId}", _userService.SignInName, _userService.IsDistributorUser, GetCurrentCorrelationId());
```

Comment:
Please update all relevant logs with similar changes.
```suggestion
                _logger.LogInformation(EventIds.Start.ToEventId(), "Maritime safety information request to get weekly NM files started for User:{SignInName} and IsDistributor:{IsDistributorUser} and correlationId:{correlationId}", _userService.SignInName ?? "Public", _userService.IsDistributorUser, GetCurrentCorrelationId());
```